### ISSUE TITLE
Impossible to use ipykernel package with VSCode Jupyter Extension

### ISSUE THREAD

**Issue by yoannbtn** [microsoft/vscode-jupyter#11845]
### Applies To

- [x] Notebooks (.ipynb files)
- [ ] Interactive Window and\/or Cell Scripts (.py files with \#%% markers)

### What happened?

```
Running cells with Python 3.9.13 ('base') requires ipykernel package.
Run the following command to install 'ipykernel' into the Python environment. 
Command: 'conda install -n base ipykernel --update-deps --force-reinstall'
```
When I run this command in the right env terminal, I have this : 

```
$ conda install -n base ipykernel --update-deps --force-reinstall
Collecting package metadata (current_repodata.json): done
Solving environment: / Solving environment\ - 

Updating ipython is constricted by 

anaconda -> requires ipython==7.31.1=py39haa95532_1

If you are sure you want an update of your package either try `conda update --all` or install a specific version of the package you want using `conda install <pkg>=<version>`
\


Updating ca-certificates is constricted by

anaconda -> requires ca-certificates==2022.07.19=haa95532_0

If you are sure you want an update of your package either try `conda update --all` or install a specific version of the package you want using `conda install <pkg>=<version>`



Updating jupyter_core is constricted by

anaconda -> requires jupyter_core==4.11.1=py39haa95532_0 

If you are sure you want an update of your package either try `conda update --all` or install a specific version of the package you want using `conda install <pkg>=<version>`



Updating certifi is constricted by

anaconda -> requires certifi==2022.9.14=py39haa95532_0   

If you are sure you want an update of your package either try `conda update --all` or install a specific version of the package you want using `conda install <pkg>=<version>`



Updating setuptools is constricted by

anaconda -> requires setuptools==63.4.1=py39haa95532_0   

If you are sure you want an update of your package either try `conda update --all` or install a specific version of the package you want using `conda install <pkg>=<version>`



Updating vc is constricted by

anaconda -> requires vc==14.2=h21ff451_1

If you are sure you want an update of your package either try `conda update --all` or install a specific version of the package you want using `conda install <pkg>=<version>`



Updating tornado is constricted by

anaconda -> requires tornado==6.1=py39h2bbff1b_0

If you are sure you want an update of your package either try `conda update --all` or install a specific version of the package you want using `conda install <pkg>=<version>`



Updating tzdata is constricted by

anaconda -> requires tzdata==2022c=h04d1e81_0

If you are sure you want an update of your package either try `conda update --all` or install a specific version of the package you want using `conda install <pkg>=<version>`



Updating jupyter_client is constricted by

anaconda -> requires jupyter_client==7.3.4=py39haa95532_0

If you are sure you want an update of your package either try `conda update --all` or install a specific version of the package you want using `conda install <pkg>=<version>`



Updating pygments is constricted by

anaconda -> requires pygments==2.11.2=pyhd3eb1b0_0       

If you are sure you want an update of your package either try `conda update --all` or install a specific version of the package you want using `conda install <pkg>=<version>`

done
done

## Package Plan ##

  environment location: C:\Users\yoann\anaconda3

  added / updated specs:
    - backcall
    - ca-certificates
    - certifi
    - colorama
    - debugpy
    - decorator
    - entrypoints
    - ipykernel
    - ipython
    - jedi
    - jupyter_client
    - jupyter_core
    - libsodium
    - matplotlib-inline
    - nest-asyncio
    - openssl
    - packaging
    - parso
    - pickleshare
    - pip
    - prompt-toolkit
    - psutil
    - pygments
    - pyparsing
    - python-dateutil
    - python=3.9
    - pywin32
    - pyzmq
    - setuptools
    - six
    - sqlite
    - tornado
    - traitlets
    - tzdata
    - vc
    - vs2015_runtime
    - wcwidth
    - wheel
    - wincertstore
    - zeromq



Proceed ([y]/n)?
```

So I press 'y' but nothing change : I have still the same error message : 

```
Running cells with Python 3.9.13 ('base') requires ipykernel package.
Run the following command to install 'ipykernel' into the Python environment. 
Command: 'conda install -n base ipykernel --update-deps --force-reinstall'
```

 I tried to create other conda env and the problem is the same. I specify that I only have this problem with VSCode, when I use Jupyter notebook directly in my browser I don't have the problem. I tried everything I found on the internet but nothing worked.

Please help me

### VS Code Version

1.72.2 (user setup)

### Jupyter Extension Version

v2022.9.1202862440

### Coding Language and Runtime Version

Python 3.9.13 (i tried .12 and the pb is the same)

### Anaconda Version (if applicable)

conda 22.9.0

**Comment by DonJayamanne** [comment]
> when I use Jupyter notebook directly in my browser I don't have the problem. I tried everything

how do you start jupyter outside vscode in The terminal. please could you provide details and the terminal instructions used.

**Comment by yoannbtn** [comment]
Well when I tried outside VSCode I was using the following commands : 
```
conda activate base
jupyter notebook
```
Then in my browser the kernel is working perfectly. I also tried to copy the URL "http://localhost:8888/?token=..." to use it as an external jupyter server and it seemed to work well. 

However, I can't get the local ipykernel worked with VSCode, I have always the same issue described above.
